guides/source/active_record_migrations.md

```diff
@@ -436,8 +436,17 @@ change_column_default :products, :approved, false
 This sets `:name` field on products to a `NOT NULL` column and the default
 value of the `:approved` field to false.
 
-TIP: Unlike `change_column` (and `change_column_default`), `change_column_null`
-is reversible.
+Additionaly, for `change_column_default`, you can pass a Hash which defines the
+changes. This will make the command reversible. The command above can be
+rewritten as:
+
+```ruby
+change_column_default :products, :approved, from: true, to: false
+```
+
+TIP: `change_column` command is always irreversible, `change_column_null` is
```

Comment:
IMO this tip is redundant especially with so many clauses, should be enough to just clearly document reversibility of each command in that command's description.